modules/meazyBidAdapter.js

```diff
@@ -0,0 +1,152 @@
+import * as utils from '../src/utils';
+import { registerBidder } from '../src/adapters/bidderFactory';
+import { BANNER } from '../src/mediaTypes';
+
+const BIDDER_CODE = 'meazy';
+const PREBID_ENDPOINT = 'rtb-filter.meazy.co';
+const SYNC_ENDPOINT = '//sync.meazy.co/sync/iframe';
+const ENDPOINT_CONFIG = {
+  defaultCurrency: ['USD'],
+  availableSize: ['300x250', '320x480', '160x600']
+};
+
+const buildURI = (pid) => {
+  return `//${PREBID_ENDPOINT}/pbjs?host=${utils.getOrigin()}&api_key=${pid}`;
+}
+
+const validateSize = (size) => {
+  return ENDPOINT_CONFIG.availableSize.indexOf(size.join('x')) !== -1;
+}
+
+const buildImpression = (bidRequest) => {
+  const impression = {
+    id: utils.getUniqueIdentifierStr(),
+    tagid: bidRequest.adUnitCode,
+    banner: {
+      format: bidRequest.sizes.map(size => ({ w: size[0], h: size[1] }))
+    }
+  };
+
+  return impression;
+}
+
+const buildItems = (bidRequests) => {
+  const imp = [];
+  const user = {};
+
+  bidRequests.forEach(bidRequest => {
+    imp.push(buildImpression(bidRequest));
+    if (utils.deepAccess(bidRequest, 'gdprConsent.gdprApplies')) {
```

Comment:
The `gdprConsent` object exists in the `bidderRequest` param of the `spec.buildRequests()` function, not the `bidRequests` param.  You'll need to pass in the `bidderRequest` param to this overall function and read that value to make this logic work.